### DISCUSSION TITLE
开启SSR后，如果在page.data.ts的里请求里bff的数据，会报错Can't call useContext out of scope

### DISCUSSION THREAD

**Discussion by CocaStar** [web-infra-dev/modern.js#6459] [Q&A]
如题，在开启SSR后，如果在page.data.ts的里请求里bff的数据，会报错Can't call useContext out of scope，
但是不开启SSR就没事，请问是什么原因，我代码里都没有使用useContext的方法，api文件夹里只使用默认的接口都会报错。
Error: Can't call useContext out of scope, make sure @modern-js/utils is a single version in node_modules，
![5851730201740_ pic](https://github.com/user-attachments/assets/58460e60-ed3d-4e51-8b9a-56a854ede831)
<img width="1125" alt="image" src="https://github.com/user-attachments/assets/1688934d-e4e6-4060-8db5-e4f8f96055d6">
<img width="806" alt="image" src="https://github.com/user-attachments/assets/0954c1fa-8300-4b3f-86a8-ab11dc148347">

**Comment by zllkjc** [comment]
我们目前没有支持在 data loader 里调用 BFF 的能力，但这在计划之中，在内部我们已经提供了一些集成与实践。
